dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.37/V2_37_35__Update_Potential_Duplicate.sql

```diff
@@ -0,0 +1,8 @@
+alter table potentialduplicate drop constraint if exists potentialduplicate_lastupdatedby_user;
+alter table potentialduplicate add column if not exists createdbyusername varchar(255);
+alter table potentialduplicate add column if not exists lastupdatebyusername varchar(255);
+update potentialduplicate set createdbyusername = u.username from potentialduplicate pd inner join users u on u.userid = pd.lastupdatedby where u.userid = potentialduplicate.lastupdatedby ;
```

Comment:
only update when column is null

Reply:
yes good point first version was that way